packages/api/src/modules/tutorials/actions/getRandomTutorial.ts

```diff
@@ -0,0 +1,9 @@
+import Text from "../../texts/types/Text";
+import knex from "../../../../db/knex";
+
+export default async () => {
+    const texts = await knex<Text>("texts").where({
+        tutorial: true
+    });
+    return texts[Math.floor(Math.random() * texts.length)];
+};
```

Comment:
Idk why we would need a random tutorial route, it would be nice to have a route to get all the tutorials sorted by difficulty.

Reply:
Figured it would be nice, idk, I copied most of the routes from texts.